STYLE.md

```diff
@@ -18,20 +24,21 @@ sentence or heading.
 - merkle (tree|root|branch|node)
 - schnorr
 - segwit
+- sybil (as in 'sybil attack')
 - taproot
 
 ### Forbidden abbreviations
 
 - Core (use Bitcoin Core)
 
-### Unintroduced abbreviations
+### Abbreviations
```

Comment:
Maybe return this to "Unintroduced abbreviations" and place both this section and "Forbidden abbreviations" under a general "Abbreviations" section that contains the sighash note.